lib/IRGen/GenReflection.cpp

```diff
@@ -466,7 +470,7 @@ class AssociatedTypeMetadataBuilder : public ReflectionMetadataBuilder {
     for (auto AssocTy : AssociatedTypes) {
       auto NameGlobal = IGM.getAddrOfFieldName(AssocTy.first);
       B.addRelativeAddress(NameGlobal);
-      addTypeRef(AssocTy.second);
+      addTypeRef(AssocTy.second, /*generic signature*/nullptr);
```

Comment:
Should this be using the conformance DC's signature?

Reply:
Good question. If the conformance is a normal conformance, I would think so, yes.

Reply:
Oof, even more to check.